### ISSUE TITLE
What is the specific version of python3？which commit or tag ?thanks. 

### ISSUE THREAD

**Issue by JunyiXie** [ColdGrub1384/Pyto#202]
I need to compare the official cpython version to see what has been modified.

**Comment by cclauss** [comment]
import sys ; print(sys.version_info)

**Comment by JunyiXie** [comment]
Only the version is not enough to determine the specific code commit

**Comment by ColdGrub1384** [comment]
I used the first 3.8 version officially released on https://www.python.org/downloads/source/. 
Python 3.8.0 - Oct. 14, 2019

**Comment by mjpieters** [comment]
> Only the version is not enough to determine the specific code commit

Yes, it is, if not compiled from an official release the patchlevel info reflects what 

> I used the first 3.8 version officially released on https://www.python.org/downloads/source/.
> Python 3.8.0 - Oct. 14, 2019

The version string suggests otherwise:

```python
>>> import sys
>>> sys.version
'3.8.0+ (default, Jun 11 2020, 14:47:24) \n[Clang 11.0.3 (clang-1103.0.32.62)]'
```

The `+` there suggests it was compiled from a git revision after 3.8.0 was tagged but before 3.8.1rc1, see [the first commit to `patchlevel.h` after 3.8.0 was released](https://github.com/python/cpython/commit/cc06217d9d2b2d9641baabc96be62a65301bbaf9#diff-b98d9817ec6a6fc9a6338b2ca4723a61).  That said, if this was compiled from a git checkout there would be a git revision listed where it says `default` (information extracted during the configure stage).
